sdk/consumption/Azure.ResourceManager.Consumption/api/Azure.ResourceManager.Consumption.netstandard2.0.cs

```diff
@@ -1,155 +1,168 @@
     }
-    public partial class BalancePropertiesAdjustmentDetailsItem
+    public partial class TenantBillingPeriodConsumptionResource : Azure.ResourceManager.ArmResource
     {
-        internal BalancePropertiesAdjustmentDetailsItem() { }
-        public string Name { get { throw null; } }
-        public decimal? Value { get { throw null; } }
+        public static readonly Azure.Core.ResourceType ResourceType;
+        protected TenantBillingPeriodConsumptionResource() { }
+        public virtual Azure.Response<Azure.ResourceManager.Consumption.Models.ConsumptionBalanceResult> GetBalance(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Consumption.Models.ConsumptionBalanceResult>> GetBalanceAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
-    public partial class BalancePropertiesNewPurchasesDetailsItem
+}
+namespace Azure.ResourceManager.Consumption.Models
+{
+    public partial class BudgetAssociatedNotification
     {
-        internal BalancePropertiesNewPurchasesDetailsItem() { }
-        public string Name { get { throw null; } }
-        public decimal? Value { get { throw null; } }
+        public BudgetAssociatedNotification(bool isEnabled, Azure.ResourceManager.Consumption.Models.NotificationAlertTriggerType @operator, decimal threshold, System.Collections.Generic.IEnumerable<string> contactEmails) { }
+        public System.Collections.Generic.IList<string> ContactEmails { get { throw null; } }
+        public System.Collections.Generic.IList<string> ContactGroups { get { throw null; } }
+        public System.Collections.Generic.IList<string> ContactRoles { get { throw null; } }
+        public bool IsEnabled { get { throw null; } set { } }
+        public Azure.ResourceManager.Consumption.Models.RecipientNotificationLanguageCode? Locale { get { throw null; } set { } }
+        public Azure.ResourceManager.Consumption.Models.NotificationAlertTriggerType Operator { get { throw null; } set { } }
+        public decimal Threshold { get { throw null; } set { } }
+        public Azure.ResourceManager.Consumption.Models.NotificationThresholdType? ThresholdType { get { throw null; } set { } }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct BillingFrequency : System.IEquatable<Azure.ResourceManager.Consumption.Models.BillingFrequency>
+    public readonly partial struct BudgetCategoryType : System.IEquatable<Azure.ResourceManager.Consumption.Models.BudgetCategoryType>
```

Comment:
Drop `Type` suffix as it's a bit redundant to `Category` and it's suggested to avoid `Type` suffix when it makes sense to remove it.

Reply:
fixed